ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bgp-base.yml

```diff
@@ -26,8 +26,26 @@ router_bgp:
       172.16.0.0/12:
       192.168.0.0/16:
         route_map: RM-FOO-MATCH
+    peer_groups:
+      baz:
+        prefix_list_in: PL-BAR-IN
+        prefix_list_out: PL-BAR-OUT
+    neighbors:
+      192.0.2.1:
+        prefix_list_in: PL-FOO-IN
+        prefix_list_out: PL-FOO-OUT
+
   address_family_ipv6:
     networks:
       2001:db8:100::/40:
       2001:db8:200::/40:
         route_map: RM-BAR-MATCH
+    peer_groups:
+      baz:
+        prefix_list_in: PL-BAR-IN
```

Comment:
please use different names between V4 and V6 in this test case, so we ensure nothing is mixed.

Reply:
Updated PR